users/details/script.js

```diff
@@ -350,6 +371,28 @@ function generateReadableDateFromSecondsTimeStamp(timeStamp) {
   //created function for readable date format
   return new Date(timeStamp * 1000).toDateString(); // new function because we are getting the value in seconds and not milliseconds
 }
+console.log(generateReadableDateFromSecondsTimeStamp('1688445662'));
+
+function generateDaysToGo(dateStr, status) {
+  const inputDate = new Date(dateStr);
+
+  const now = new Date();
+  const offset = 330 * 60 * 1000;
+  const currentDate = new Date(now.getTime() + offset);
+  const diff = inputDate - currentDate; // Calculates the difference in milliseconds
+  if (diff <= 0 && status == 'COMPLETED') {
+    return 'Task Completed Within Deadline';
+  } else if (diff <= 0) {
+    return 'Deadline Passed'; // Due date is in the past
+  } else if (diff > 0 && diff < 24 * 60 * 60 * 1000) {
+    return 'Less Than a Day Remaining'; // Less than a day remaining
+  } else {
+    const daysRemaining = Math.floor(diff / (24 * 60 * 60 * 1000)); // Calculate the days remaining
+    return daysRemaining === 1
+      ? '1 day remaining'
+      : daysRemaining + ' days remaining'; // Handle singular and plural for 1 day and more than 1 day
```

Comment:
Day 'D' upper case please

Reply:
Implemented
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/31709147/8e62ffc7-2d54-4f7d-983b-e080cf70822b)

Reply:
missed one spot, updating it as well

Reply:
Updated.
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/31709147/54edfa53-f18d-4753-9b51-d636475cc130)